src/vivarium/framework/components/manager.py

```diff
@@ -70,19 +89,54 @@ def setup_components(self, builder, configuration):
 
 
 class ComponentInterface:
+    """The component manager interface made available from the builder."""
 
     def __init__(self, component_manager: ComponentManager):
         self._component_manager = component_manager
 
     def add_components(self, components: Sequence):
+        """Register new components with the system. Components are setup after
+        managers.
+
+        Parameters
+        ----------
+        components:
+          Instantiated components to register
+        """
         self._component_manager.add_components(components)
 
     def get_components(self, component_type: str):
         return self._component_manager.get_components(component_type)
 
 
 def _setup_components(builder, component_list: Sequence, configuration: ConfigTree) -> Sequence:
-
+    """Helper method for configuring and setting up a list of components.
```

Comment:
Let's make all these methods public.  Otherwise they won't render in the api docs